### PR TITLE
Renames JadeSourceFileParser to JavaSourceFileParser

### LINKED ISSUE #7: JadeSourceFileParser should be renamed JavaSourceFileParser
@petroseskinder you're right :)

### PR DESCRIPTION
**Pull request by petroseskinder**
Resolves #7

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -40,3 +40,6 @@ hs_err_pid*
 # Ignore jekyll build output.
 /production
 /.sass-cache
+
+# Ignore mac attributes files
+*.DS_STORE
```
Comment (cgrushko) [inline .gitignore:45]: this is unrelated to the subject of the PR; how strict is it in OSS to require separate PRs for separate things?

**Comment by cgrushko** [comment]
I tried adding a webhook for Jenkins, let's see if it triggers.

**Comment by petroseskinder** [comment]
From what I have seen, it's not that strict. I was also surprised by that. There are two levels of granularity. The commit level and the pull request. When you ask for changes, I would make another commit. Then finally, when things are merged only the pull request displays.

**Comment by cgrushko** [comment]
Ok. Let's leave it this time, I'll gradually become more strict :)

**Comment by cgrushko** [comment]
retest this please

**Comment by cgrushko** [comment]
test this please
